core/frontend/package.json

```diff
@@ -3,6 +3,46 @@
   "version": "0.1.0",
   "private": true,
   "vue": {
+    "devServer": {
```

Comment:
How should I use it?

Reply:
yarn run serve will use this

Reply:
then you should be able to open localhost:8080 and everything should work automagically.
you can delete some of this lines or point it to your machine if you want to run some apis locally.

Reply:
Cool!